test/built-ins/Temporal/PlainYearMonth/prototype/add/calendar-datefromfields-called.js

```diff
@@ -77,7 +77,6 @@ class CustomCalendar extends Temporal.Calendar {
     return new Temporal.PlainMonthDay(isoMonth, isoDay, this, isoYear);
   }
   dateAdd(date, duration, options) {
-    if (duration.months) throw new Error("adding months not implemented in this test");
```

Comment:
We would probably want to implement this properly rather than mixing ISO 8601 dateAdd with the weird month scheme in this test calendar.

Off the top of my head, you can probably do it something like this:
```js
const {isoYear, isoMonth, isoDay} = date.getISOFields();
let {years, months, weeks, days} = duration;
let iter = new Temporal.PlainDate(isoYear + years, isoMonth, isoDay, "iso8601");
const monthsDays = months * 36;
if (iter.dayOfYear + monthsDays > iter.daysInYear || iter.dayOfYear + monthsDays < 1)
  throw new Error("complicated addition not implemented in this test");
return iter.add({ weeks, days: monthsDays + days }).withCalendar(this);
```

Reply:
The code off the top of your head seems to work :)